dashboard/app/models/user.rb

```diff
@@ -1413,7 +1413,7 @@ def self.track_level_progress_sync(user_id:, level_id:, script_id:, new_result:,
       end
 
       # Update user_level with the new attempt.
```

Comment:
nit: I'd like to see this comment updated to explain the thought behind the `unless` condition